recommender/engine/rl_agent/action_selector.py

```diff
@@ -18,24 +50,64 @@ def __call__(self, weights: torch.Tensor, user: User, search_data: SearchData) -
             weights: Weights returned by an actor model, tensor of shape [K, SE], where:
                 - K is the number of services required for the recommendation
                 - SE is a service content tensor embedding dim
-            user: User for whom the recommendation is generated. User's accessed_services is used for narrowing the itemspace down.
```

Comment:
To obtain all ordered services, both user.accessed_services and user actions (of the same user) should be used (there can be iinformation about ordering that is not avalilable in the user.accessed_services - that comes from DB dump). Appropriate function will be implemented in [Services History Generator](https://github.com/cyfronet-fid/recommender-system/blob/e8ade68838aa3d5a11e7606acddab26539e50af5/recommender/services/services_history_generator.py) according to this issue #83

Reply:
I have to wait for this PR then